### ISSUE TITLE
Nothing in the 4.x series has been published

### ISSUE THREAD

**Issue by jagthedrummer** [alienfast/middleman-aws#5]
Just followed the instructions in the `README` and when I try to `bundle install` I get:

```
Could not find gem 'middleman-aws (>= 4.0.0)' in any of the gem sources listed in your Gemfile or available on this machine.
```

And I don't find anything in the 4.x series listed here: https://rubygems.org/gems/middleman-aws

**Comment by rosskevin** [comment]
This is true.  I was tired of waiting so I've been using master.  I'll check again and try to get a build published when I have time.  For now, you can use master as well.
